lib/interface/cli/commands/trigger/delete.cmd.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('debug')('codefresh:cli:delete:trigger');
+const Command = require('../../Command');
+const CFError = require('cf-errors');
+const _ = require('lodash');
+const { trigger } = require('../../../../logic').api;
+const deleteRoot = require('../root/delete.cmd');
+
+const command = new Command({
+    command: 'trigger <event-uri> <pipeline>',
+    aliases: ['t'],
+    parent: deleteRoot,
+    cliDocs: {
+        description: 'Remove trigger from specified pipeline, this may also remove trigger event w/out pipelines too.',
+    },
+    webDocs: {
+        category: 'Triggers',
+        title: 'Remove trigger from specified pipeline',
```

Comment:
Delete Context

Reply:
Context? Do you mean "Delete trigger" instead of "Remove trigger"?